cmd/pdc/main.go

```diff
@@ -64,6 +65,13 @@ func (mf *mainFlags) RegisterFlags(fs *flag.FlagSet) {
 	fs.BoolVar(&mf.DevMode, "dev-mode", false, "[DEVELOPMENT ONLY] run the agent in development mode")
 }
 
+func (mf *mainFlags) ApplyEnvironment(grafanaPDCEnvVarPrefix string) {
+	clusterEnvVar, ok := os.LookupEnv(fmt.Sprintf("%v_CLUSTER", grafanaPDCEnvVarPrefix))
```

Comment:
For strings, you can use `%s`
```suggestion
	clusterEnvVar, ok := os.LookupEnv(fmt.Sprintf("%s_CLUSTER", grafanaPDCEnvVarPrefix))
```

This applies to all other `Sprintf` calls here